hal/targets.json

```diff
@@ -1628,6 +1628,7 @@
         "extra_labels": ["RENESAS", "MBRZA1H"],
         "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
         "inherits": ["Target"],
+        "macros": [" MBEDTLS_ENTROPY_HARDWARE_ALT "],
```

Comment:
Nit pick: Is there a reason the extra "space" characters (` `) are needed on both sides of `MBEDTLS_ENTROPY_HARDWARE_ALT`? Could the line be changed from:

```
        "macros": [" MBEDTLS_ENTROPY_HARDWARE_ALT "],
```

to

```
        "macros": ["MBEDTLS_ENTROPY_HARDWARE_ALT"],
```